src/containers/pages/OrganizationViews/helpers/tests/servicehooks.test.ts

```diff
@@ -47,9 +47,7 @@ describe('src/containers/pages/OrganizationViews/helpers/servicehooks', () => {
       mockClass,
       fetchPractitionerRolesMock,
       fetchPractitionersMock
-    ).catch(e => {
-      throw e;
-    });
+    ).catch(err => err);
```

Comment:
Do we want to use `displayError` here?

Reply:
Naah!, this is part of the test, I want the error to actually fail the test.